spec/requests/join_lab_measurement_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
```

Comment:
Saran dari mentor kemarin, nice to have kalo spec/request dipindah ke folder spec/controller dan namanya ngikut nama file controller-nya dikasih suffix _spec. Biar seragam semua penamaan folder/file nya